File: 240110/함수를 이용한 소수 판별/decimal-decisions-using-functions.java
import java.util.Scanner;

public class Main {
    public static int sosu(int z,int x){
        int tal = 0 ;
        boolean so = false;
        for (int i = z ; i<=x; i++){
            for(int j=2; j<z; j++){
                if(i%j == 0){
                    so = true;
                }
            }
            if (so == false){
                tal+=i;
            }
            so = false;
        }
        return tal;
    }
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        System.out.print(sosu(a,b));
    }
}

File: 240110/함수를 이용한 소수 판별/decimal-decisions-using-functions.py
a,b = map(int,input().split())

def sosu(i):
    
    for k in range(2,i):
        if i% k == 0:
            return False
    return True

tal = 0 

for i in range(a,b+1):
    if sosu(i):
        tal = tal + i 
print(tal)